scripts/run-integration-test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Is it just to simulate locally what the GitHub action is doing? I take `go test -v -tags=integration ./pkg/bitcoin/electrum` should work as well.

Reply:
Yes, but I wanted to run them 1:1 as CI. Also running them in docker may make it simpler for contributors, as it won't require full local dev env setup.